c2corg_api/search/mappings/report_mapping.py

```diff
@@ -0,0 +1,54 @@
+from c2corg_api.models.report import REPORT_TYPE, Report
+from c2corg_api.search.mapping import SearchDocument, BaseMeta
+from c2corg_api.search.mapping_types import QueryableMixin, QEnum, \
+  QEnumArray, QInteger, QDate
+
+
+class SearchReport(SearchDocument):
+    class Meta(BaseMeta):
+        doc_type = REPORT_TYPE
+
+    # TODO - ADD OTHER FIELDS - names from
+    # https://github.com/c2corg/camptocamp.org/blob/
+    # 72e777075f2e6260b1d2ef563ab0f90c383038ba/apps/frontend/modules/
+    # xreports/config/module.yml
+
+    activities = QEnumArray(
+        'act', model_field=Report.activities)
+    date = QDate('xdate', 'date')
+    event_type = QEnum(
+         'xtyp', model_field=Report.event_type)
+    nb_participants = QEnum(
```

Comment:
For the fields nb_participants, avalanche_slope and others I makes probably sense to use `QEnumRange` (see also: https://github.com/c2corg/v6_api/pull/327). But we can start with a minimum of search fields in this PR.